src/ol/render/webgl/BatchRenderer.js

```diff
@@ -89,11 +89,12 @@ class AbstractBatchRenderer {
    * @param {import("./MixedGeometryBatch.js").GeometryBatch} batch Geometry batch
    * @param {import("../../transform.js").Transform} currentTransform Transform
    * @param {import("../../PluggableMap.js").FrameState} frameState Frame state.
+   * @param {number} [offsetX] Optional x offset
    */
-  render(batch, currentTransform, frameState) {
+  render(batch, currentTransform, frameState, offsetX) {
     // multiply the current projection transform with the invert of the one used to fill buffers
     // FIXME: this should probably be done directly in the layer renderer
-    this.helper_.makeProjectionTransform(frameState, currentTransform);
+    this.helper_.makeProjectionTransform(frameState, currentTransform, offsetX);
```

Comment:
```suggestion
    this.helper_.makeProjectionTransform(frameState, currentTransform);
    translate(currentTransform, offSetX);
```

The offsetX parameter could be removed from makeProjectionTransform

Reply:
Thanks, that makes so much sense... I'll add another commit to add this and revert the helper, and I guess we can rebase everything afterwards!